install.sh

```diff
@@ -4,37 +4,54 @@
 
 #This assumes all is installed under /opt/HardwareCheckout
 
-APP_PATH=/opt/hardware
-GIT_PATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+GITREPO=https://github.com/linted/HardwareCheckout.git
+APP_PATH=/opt/HardwareCheckout
 DBNAME=chvappdb
 DBUNAME=chvapp
 DBPASS=$(head -10 /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | sort -r | head -n 1)
 
+echo "Run this once and you should be set... Make sure to run it as root!"
+
+
+if [ ! -d "$APP_PATH" ]; then
+
+  while true; do
+    read -p "$APP_PATH does not exist - do you want me to clone the repo? (y/N)?" yn
+    case $yn in
+        [Yy]* ) sudo git clone $GITREPO $APP_PATH; break;;
```

Comment:
This should not be sudo. That will set the contents of the cloned repo to be two strong of permissions. Instead we should make the directory with sudo then change the permissions to allow the current user to clone into it.

Reply:
Workflow logic here was not sound anyway; removed...